main/conceptual/js-programming/eventual-send.md

```diff
@@ -25,15 +25,15 @@ E(zoe).install(bundle)
 
 ## Eventual Send
 
-One of the ways [Zoe partitions risk](https://www.youtube.com/watch?v=T6h6TMuVHKQ&t=368s) is by running in its own vat, separate from any smart contract that might
+One of the ways [Zoe partitions risk](https://www.youtube.com/watch?v=T6h6TMuVHKQ&t=368s) is by running in its own [vat](../../glossary/README.md#vat), separate from any smart contract that might
 use too much compute time or heap space. The smart contracts also run in separate vats.
 
-![Zoe in a separate vat](../../assets/zoe-partitions-risk-slide.svg)
+![Zoe in a separate vat](../assets/zoe-partitions-risk-slide.svg)
 
 What happens when we call `E(zoe).install(bundle)` is an _eventual send_:
 
- 1. A message consisting of a the method name `install`
-    with the arguments structure `[bundle]` is [marshaled](/guides/js-programming/far.md)
+ 1. A message consisting of a method named `install`
+    with the `bundle` argument is [marshaled](./far.md)
```

Comment:
no, the message doesn't contain a method (that would be a function). It contains a method _name_; that is: a string.

And what gets marshaled is a list... but maybe that part is clear enough either way.

Reply:
Right you are. I changed the text to "A message consisting of the method name install with the bundle argument marshaled to a flat string and queued for delivery to the vat that zoe comes from." It's still not quite correct, since it gets marshaled to a series of strings, not "a flat string", exactly. But it's better than what I had, and no worse than what was there previously, so it's good enough for now.